frontend/src/scenes/session-recordings/player/PlayerMetaLinks.tsx

```diff
@@ -29,6 +31,23 @@ export function PlayerMetaLinks(): JSX.Element {
         return Math.floor(playerTime / 1000)
     }
 
+    const commentInExistingNotebook = useCallback(
+        (notebookShortId) => {
+            const currentPlayerTime = getCurrentPlayerTime() * 1000
+            openNotebook(notebookShortId, NotebookTarget.Popover)
+            setTimeout(() => {
+                if (!nodeLogic) {
+                    throw new Error('wat')
+                }
+                // need time for the notebook to open and mount its logic
```

Comment:
Mmm I think we'll face this in a good number of places. Don't know exactly how we'll get around it but certainly feels like something to be handled at a notebooks platform level